web/src/components/dashboard/ProjectRequests.tsx

```diff
@@ -161,7 +161,7 @@ const ProjectRequests: React.FC<any> = (props) => {
           />
         }
       />
-      <Box style={{ overflow: 'auto' }}>
```

Comment:
Does the request table also need to have this change? does it have the same problem?

Reply:
I changed the request table as well to have consistency within the two tables. If the request table were ever to gain more columns, then it will have the same behaviour as the project details table.